Public/Sdk/Public/Managed/helpers.dsc

```diff
@@ -132,4 +136,44 @@ namespace Helpers {
 
         return results.toArray();
     };
+
+    /** TODO: Unofortunately the System.Interactive.Async package exposes IAsyncEnumerable in its own System.Generic.Collection namespace,
+    /*        colliding with .NETCore 3.0 implementation of the same name. For the time being (hopefully once the package maintainers updated their deps),
+    /*        we explicitly create an aliased reference for the code that depends on this.
+    **/
+
+    @@public
+    export function aliasedReferencesForSystemInteractiveAsync(references: Shared.Reference[]) : Csc.Arguments {

```

Comment:
>aliasedReferencesForSystemInteractiveAsync [](start = 20, length = 42)

Nit: Rename to indicate it is a hack to patch the referenes: i.e. `patchReferencesFor...` or `hackReferencesFor...`